catalog/ui/search-ui/standard/pom.xml

```diff
@@ -214,9 +214,16 @@
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctor-maven-plugin</artifactId>
                 <version>1.5.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.asciidoctor</groupId>
+                        <artifactId>asciidoctorj-pdf</artifactId>
+                        <version>1.5.0-alpha.6</version>
```

Comment:
In the dib repo we have root pom level version variables that we use for the plugin and the asciidoctor dependencies... In ddf we only have the one inside the distribution/docs/pom.xml... should we create a ticket to have these at the ddf root pom level?